src/app/feature-flag/notification-footer/feature-footer.component.html

```diff
@@ -3,26 +3,25 @@
   <div *ngSwitchCase="'internal'">
     <ng-template #tooltipTemplateInternal>
       <div>
-          <span *ngIf="featurePageConfig?.featuresPerLevel?.internal?.length !== 0">
-            <strong>There are {{featurePageConfig?.featuresPerLevel?.internal?.length}} internal feature(s) on this page.</strong>
-            <ul>
-              <li *ngFor="let feature of featurePageConfig?.featuresPerLevel?.internal">
-                <span class="pficon pficon-info"></span>
-                {{feature.attributes.name}}
-              </li>
-            </ul>
-          </span>
+        <span *ngIf="featurePageConfig?.featuresPerLevel?.internal?.length !== 0">
+          <strong>There {{internalFeatureText}} on this page.</strong>
+          <ul>
+            <li *ngFor="let feature of featurePageConfig?.featuresPerLevel?.internal">
+              <span class="pficon pficon-info"></span>
+              {{feature.attributes.name}}
+            </li>
+          </ul>
+        </span>
         <span *ngIf="featurePageConfig?.featuresPerLevel?.experimental?.length !== 0">
-            <strong>There are {{featurePageConfig?.featuresPerLevel?.experimental?.length}} experimental feature(s) on this page.</strong>
-            <ul>
+             <strong>There {{experimentalFeatureText}} on this page.</strong>            <ul>
```

Comment:
Above you fixed the indentation of tags to be two spaces, but it looks like 4 spaces here still